packages/client/modules/email/components/SummaryEmail/MeetingSummaryEmail/WholeMeetingSummary.tsx

```diff
@@ -33,20 +39,35 @@ const WholeMeetingSummary = (props: Props) => {
             }
           }
         }
+        ... on TeamPromptMeeting {
+          responses {
+            id
+          }
+        }
       }
     `,
     meetingRef
   )
-  if (meeting.__typename !== 'RetrospectiveMeeting') return null
-  const {summary: wholeMeetingSummary, reflectionGroups, phases} = meeting
-  const discussPhase = phases.find((phase) => phase.phaseType === 'discuss')
-  const {stages} = discussPhase ?? {}
-  const hasTopicSummary = reflectionGroups.some((group) => group.summary)
-  const hasDiscussionSummary = !!stages?.some((stage) => stage.discussion?.summary)
-  const hasOpenAISummary = hasTopicSummary || hasDiscussionSummary
-  if (!hasOpenAISummary) return null
-  if (hasOpenAISummary && !wholeMeetingSummary) return <WholeMeetingSummaryLoading />
-  return <WholeMeetingSummaryResult meetingRef={meeting} />
+  if (meeting.__typename === 'RetrospectiveMeeting') {
+    const {summary: wholeMeetingSummary, reflectionGroups, phases} = meeting
+    const discussPhase = phases!.find((phase) => phase.phaseType === 'discuss')
+    const {stages} = discussPhase ?? {}
+    const hasTopicSummary = reflectionGroups!.some((group) => group.summary)
+    const hasDiscussionSummary = !!stages?.some((stage) => stage.discussion?.summary)
+    const hasOpenAISummary = hasTopicSummary || hasDiscussionSummary
+    if (!hasOpenAISummary) return null
+    if (hasOpenAISummary && !wholeMeetingSummary) return <WholeMeetingSummaryLoading />
```

Comment:
+1 i could be wrong here, but it looks like summary could be null if that fetch fails (e.g. openAI is down for a bit). Is there a way we could distinguish between an error state vs. a perpetual loading state?

Reply:
This is accurate, but this is just the current state of this component (not something new in this PR), so I'd prefer to tackle that separately.

I've created https://github.com/ParabolInc/parabol/issues/8615 to track this